tileview/src/main/java/com/qozix/tileview/tiles/Tile.java

```diff
@@ -255,7 +258,12 @@ void reset() {
     mState = State.UNASSIGNED;
     mRenderTimeStamp = null;
     if( mBitmap != null && !mBitmap.isRecycled() ) {
-      mBitmap.recycle();
+      BitmapRecycler recycler = mBitmapRecyclerReference.get();
+      if( recycler != null ) {
+        recycler.recycleBitmap( mBitmap );
+      } else {
```

Comment:
I can imagine some conditions in which you could end up in the odd position of having garbage collected the `recycler` before the `Tile` gets to use it. The `else` block, as it is, errs on the side of safety in that failure mode/edge case. If you disagree, I'll remove it.